apps/enc.c

```diff
@@ -570,7 +570,8 @@ int enc_main(int argc, char **argv)
     BIO_free(bzl);
 #endif
     release_engine(e);
-    OPENSSL_free(pass);
+    if (pass != NULL)
+        OPENSSL_clear_free(pass, strlen(pass));
```

Comment:
Well, one can argue that cleansing around line 627 is conditional, i.e. that there are cases when data won't be zeroed. So that this modification can be viewed as one *ensuring* that data is cleaned. But then it might be appropriate to remove cleansing around line 627...